packages/gatsby/.npmignore

```diff
@@ -1,3 +0,0 @@
-src
```

Comment:
Did you intend to drop the .npmignore file entirely? Or even change it?

Reply:
yes I do cause it's not used, we use the files directive in package.json